Added Circle Function

davemk00 opened the pull request:
Added another checkbox, and functionality to "Circularize" triangle features. Pretty straightforward replication of the Square functionality throughout
- Some minor tweaks to text, format and "X" in remove row button.
  I tried to adjust the "delete row" button was at the right most, but couldn't so I gave up.

davemk00 committed:
Added Circle Function

Added another checkbox, and functionality to "Circularize" triangle
features.
- Some minor tweaks to text, format and "X" in remove row button

their commented:
Looks great!
However, before going forward: It looks like the functionality stopped working at some point. It was working up to a few weeks ago when last test. So some time in between then and now the original functionality (squaring, adding tag) stopped working. Might need to fix this before adding your new functionality. Anyway, looks great what you have done.

davemk00 commented:
Glad to add to your project.
btw, I haven't had any issues with the squaring, tagging, ctrl, etc. Seems to be working fine on my system.

davemk00 committed:
Added Line function

- Added "Line" function. Similar to the Area next exec method.
- Added mutually exclusivity between pairs: Area/Line  and  also
Square/Circle.  When line is selected, Square/Circle are deselected. I'm
really not sure if it's a good/correct way to do the mutual exclusivity,
but it works so....
- tweaks to the hotkey menu. Paired Line/Area, and Square/Circle.
Changed widths.
